### ISSUE TITLE
readthedocs build fails but does not throw an error

### ISSUE THREAD

**Issue by agoscinski** [lab-cosmo/equistore#192]
Looking at the output of any open PR we can see that the readthedocs build fails when building it (last command) but it does not return any error. For example https://readthedocs.org/projects/equistore/builds/19725743/ That is probably why https://github.com/lab-cosmo/equistore/pull/184 does not show any information on the doc page

Here is error message.
```
WARNING: autodoc: failed to import class 'TensorBlock' from module 'equistore'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/equistore/envs/170/lib/python3.10/site-packages/sphinx/ext/autodoc/importer.py", line 60, in import_module
    return importlib.import_module(modname)
  File "/home/docs/.asdf/installs/python/3.10.8/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/docs/checkouts/readthedocs.org/user_builds/equistore/checkouts/170/python/src/equistore/__init__.py", line 18, in <module>
    from .version import __version__  # noqa
  File "/home/docs/checkouts/readthedocs.org/user_builds/equistore/checkouts/170/python/src/equistore/version.py", line 5, in <module>
    __version__ = get_distribution("equistore").version
  File "/home/docs/checkouts/readthedocs.org/user_builds/equistore/envs/170/lib/python3.10/site-packages/pkg_resources/__init__.py", line 466, in get_distribution
    dist = get_provider(dist)
  File "/home/docs/checkouts/readthedocs.org/user_builds/equistore/envs/170/lib/python3.10/site-packages/pkg_resources/__init__.py", line 342, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/home/docs/checkouts/readthedocs.org/user_builds/equistore/envs/170/lib/python3.10/site-packages/pkg_resources/__init__.py", line 886, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/docs/checkouts/readthedocs.org/user_builds/equistore/envs/170/lib/python3.10/site-packages/pkg_resources/__init__.py", line 772, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'equistore' distribution was not found and is required by the application
```

**Comment by Luthaf** [comment]
Good catch. This is some fallout from #85, where I added `__version__`. I'll check how we can fix this!

**Comment by agoscinski** [comment]
could it be that
```
python -m sphinx -T -E -b html -d _build/doctrees -D language=en . $READTHEDOCS_OUTPUT/html
```
should be
```
python -m sphinx -T -E -b html -d user_builds/doctrees -D language=en . $READTHEDOCS_OUTPUT/html
```
?

**Comment by agoscinski** [comment]
seems like `{posargs:-E}`
https://github.com/lab-cosmo/equistore/blob/16ff8ea4b1b466892473e53bf042e1683417e410/tox.ini#L77
is not doing its job

**Comment by agoscinski** [comment]
Okay talked with @Luthaf was just the .readthedocs.yml that was missing installation of equistore

**Comment by Luthaf** [comment]
Fixed in d77b01babe165dfe378ececc322a2fb0af0d4235
